### ISSUE TITLE
React 18.3.1 couldn't resolve as dependency 

### ISSUE THREAD

**Issue by MiXeR54** [Telegram-Mini-Apps/TelegramUI#39]
Can we bump React dependant version?

```
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: reactjs-template@0.0.1
npm error Found: react@18.3.1
npm error node_modules/react
npm error   react@"18.3.1" from the root project
npm error
npm error Could not resolve dependency:
npm error peer react@"18.2.0" from @telegram-apps/telegram-ui@2.1.4
npm error node_modules/@telegram-apps/telegram-ui
npm error   @telegram-apps/telegram-ui@"2.1.4" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by MiXeR54** [comment]
I've made changes for this issue, also update all deps in repo.

Please review.

https://github.com/Telegram-Mini-Apps/TelegramUI/pull/40

**Comment by mainsmirnov** [comment]
Thank you, fixed in version `2.1.5`
